homework2/Program.cs

```diff
@@ -8,8 +8,49 @@ namespace homework2
 {
     class Program
     {
+        enum TransactionType
+        {
+            Buy = 1,
+            Sell = -1
+        }
+
         static void Main(string[] args)
         {
+
+            Console.WriteLine("Input the nominal of the trade");
+            var UInominal = Console.ReadLine();
```

Comment:
please try use same naming convention. UInominal - in PascalCasing and other variables in camelCasing. Consider using camelCasing to all variable names. You can change UInominal to userInput or nominalInput